registrations/views.py

```diff
@@ -15,195 +16,31 @@
-        'redcross.lv',
-        'redcross.md',
-        'redcross.mn',
-        'redcross.no',
-        'redcross.or.ke',
-        'redcross.or.kr',
-        'redcross.or.th',
-        'redcross.org',
-        'redcross.org.au',
-        'redcross.org.ck',
-        'redcross.org.cn',
-        'redcross.org.cy',
-        'redcross.org.hk',
-        'redcross.org.jm',
-        'redcross.org.kh',
-        'redcross.org.lb',
-        'redcross.org.mk',
-        'redcross.org.mm',
-        'redcross.org.mo',
-        'redcross.org.mt',
-        'redcross.org.mz',
-        'redcross.org.na',
-        'redcross.org.nz',
-        'redcross.org.pg', # Papua New Guinea
-        'redcross.org.ph',
-        'redcross.org.pg',
-        'redcross.org.rs',
-        'redcross.org.sb', # Solomon Islands
-        'redcross.org.sg',
-        'redcross.org.ua',
-        'redcross.org.uk',
-        'redcross.org.vn',
-        'redcross.org.za',
-        'redcross.org.zm',
-        'redcross.ru',
-        'redcross.se',
-        'redcross.sk',
-        'redcross.tl',
-        'redcrossghana.org',
-        'redcrosseth.org',
-        'redcrossmuseum.ch',
-        'redcrossnigeria.org',
-        'redcrossseychelles.sc',
-        'redcrossug.org',
-        'redcrossvanuatu.com',
-        'redcrosszim.org.zw',
-        'rks.si',
-        'rmiredcross.org',
-        'rodekruis.be',
-        'roteskreuz.be',
-        'rodekors.dk',
-        'rodekruis.nl',
-        'redcross.nl',
-        'roteskreuz.at',
-        'st.roteskreuz.at', # Steiermark (Styria)
-        's.roteskreuz.at',  # Salzburg
-        'b.roteskreuz.at',  # Burgenland
-        't.roteskreuz.at',  # Tirol (Tyrol)
-        'w.roteskreuz.at',  # Wien (Vienna)
-        'v.roteskreuz.at',  # Vorarlberg
-        'k.roteskreuz.at',  # Kaernten (Carinthia)
-        'n.roteskreuz.at',  # Niederoesterreich (lower Austria)
-        'o.roteskreuz.at',  # Oberoesterreich (upper Austria)
-        'roteskreuz-innsbruck.at',
-        'roteskreuz.li',
-        'rwandaredcross.org',
-        'sarc.sy',
-        'sarc-sy.org',
-        'sierraleoneredcross.org',
-        'srcs.sd',
-        'standcom.ch',
-        'tgymj.gov.tm',
-        'tongaredcross.to',
-        'trcs.or.tz',
-        'ttrcs.org',
-        'vanuaturedcross.org',
-        'voroskereszt.hu' # no comma
+    allowed_list = [dom for dom in 
+        (
+            DomainWhitelist.objects
+                .filter(is_active=True)
+                .values_list('domain_name', flat=True)
+                .annotate(domain_name_lower=Lower('domain_name'))
+        )
     ]
+    
+    # Adding ifrc.org as a trusted domain always
+    if not allowed_list or 'ifrc.org' not in allowed_list:
+        allowed_list.append('ifrc.org')
+    # These 2 are needed for test_views.py
+    if 'arcs.org.af' not in allowed_list:
+        allowed_list.append('arcs.org.af')
+    if 'voroskereszt.hu' not in allowed_list:
+        allowed_list.append('voroskereszt.hu')
```

Comment:
@GregoryHorvath It will be a good idea to add fixtures to load previous static data and some for testing (remove this in production manually).

Reply:
@thenav56 I see what you mean, though those are part of the list anyway. It's just they were needed for the build to be even successful by CircleCI.

Reply:
Okay.
PS: You can use also `in` eg: ` if is_allowed or domain in ['ifrc.org', 'arcs.org.af', 'voroskereszt.hu']:`

Reply:
Wait sorry I missed to answer the real point of your first comment. So this is there just if anyone would happen to delete all the records in that table somehow it would still allow 'ifrc.org' for example so one could register that way. Same why I did a similar thing for the frontend validation.